hkmc2/shared/src/main/scala/hkmc2/semantics/Resolver.scala

```diff
@@ -633,6 +647,7 @@ class Resolver(tl: TraceLogger)
           
           val args = as match
             case Term.Tup(args) => args
+            case Term.CtxTup(args) => args
             case spd => Spd(true, spd) :: Nil
```

Comment:
Is this exhaustive? Is `spd` really a spread? Are error cases potentially missing, here?

Reply:
I remember I saw a comment in the code base saying any `Term` occurring as the RHS of an `App` is a spread. Let me find it out

Reply:
Here it is

https://github.com/hkust-taco/mlscript/blob/f2c2999f96f4a3f4299256f0d6eb82142fb68584/hkmc2/shared/src/main/scala/hkmc2/codegen/Lowering.scala#L325-L332

Reply:
Ah, then please document that here too!